src/nnsight/contexts/Tracer.py

```diff
@@ -179,6 +180,9 @@ def remote_backend_handle_result_value(self, value: Dict[str, Any]) -> None:
         # TODO : graph mismatch handle. hash json ?
         for node_name, node_value in value.items():
             self.graph.nodes[node_name]._value = node_value
+            
+    def remote_backend_get_stream_node(self, name: str, graph_id: str) -> "Node":
```

Comment:
I get that the function name provides context for when the function is called (and perhaps the only actual usecase), but why not use a more general name, like `get_node()`? It is just a getter function afterall